Tests/IntegrationTests.Shared/PrimaryKeyTests.cs

```diff
@@ -275,6 +275,27 @@ public void PrimaryKeyFailsIfClassNotinRealm()
                 var obj = skinny.ObjectForPrimaryKey<PrimaryKeyInt64Object>(42);
             });
         }
+
+        // mostly a little test to show iterating schema and getting primary key names
+        [Test]
+        public void CanGetNameOfPrimaryKeyFromSchema()
+        {
+            int classesWithPK = 0;
+            int classesWitoutPK = 0;
+            foreach (var objSchema in _realm.Schema)
+            {
+                var primaryKeyProp = objSchema.Where(prop => prop.IsPrimaryKey).FirstOrDefault();
+                if (string.IsNullOrEmpty(primaryKeyProp.Name))
+                    classesWitoutPK++;
+                else
+                {
+                    classesWithPK++;
+                    Debug.WriteLine($"PK test - {objSchema.Name} has a PK {primaryKeyProp.Name}");
+                }
+            }
+            Assert.That(classesWitoutPK > classesWithPK);  // slightly crap assertion to ensure test works
```

Comment:
Yes, that might be a nicer unit test but I wanted this to be fairly independent of changes to all our other class declarations and, most importantly, illustrate the point of iterating through a bunch of classes and pulling out those with primary keys.

It would get larger and noisier to add the PK/class matches.